gef-ui/server/utils/helpers.js

```diff
@@ -487,6 +487,15 @@ const pastDate = ({ day, month, year }) => {
   return input < now;
 };
 
+// checks that both dates are the same
+const sameDate = ({ day, month, year }, coverEndDate) => {
+  const input = getEpoch({ day, month, year });
```

Comment:
I've seen that you the function is being called using `details?.coverEndDate` <- this can be undefined.
If that's the case, do we need to add a validation first that checks whether coverEndDate has a value?

Reply:
We do need `coverEndDate` null validation please.

Reply:
done - though can never be null